network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/tito_widget_block.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./base_streamfield_block.html" %}
+
+{% block block_content %}
+    <script src='https://js.tito.io/v2' async></script>
+
+    {% if value.redirect_page %}
+        <script>
+        // This copes with the javascript being loaded
+        // asycronously (recommended)
+        window.tito =
+            window.tito ||
+            function() {
+            (tito.q = tito.q || []).push(arguments);
+            };
+
+        // For example:
+        tito('on:registration:finished', function(data){
+            window.location.assign("{{ value.redirect_page.full_url }}");
+        })
+        </script>
```

Comment:
Rather than inlining this, let's instead make this part of the mozfest JS bundle (entry point: `source/js/foundation/pages/mozfest/index.js`), and then with that `{{ value.redirect_page.full_url }}` as content in a queryable element (e.g. `<input type="hidden" name="tito-post-registration-url" value="{{ value.redirect_page.full_url }}">`).

So have this in a file like `.../mozfest/tito.js`

```javascript
export function setupTito() {
  globalThis.tito = globalThis.tito || function() {
    (tito.q = tito.q || []).push(arguments);
  };

  tito('on:registration:finished', function(data) {
    const redirectURL = document.querySelector(`input[name="tito-post-registration-url"]`)?.value;
    if (redirectURL) {
      globalThis.location = redirectURL;
    }
  });
};
```
and then import and run that in the `index.js` file:

```javascript
import { bindEventHandlers } from "./template-js-handler";
import { setupTito } from "./tito.js";

bindEventHandlers();
setupTito();
```